superdesk/editor_utils.py

```diff
@@ -819,3 +820,15 @@ def render_fragment(elem) -> str:
         # client renders empty paragraph as `<p><br></p>`
         etree.SubElement(elem, "br", nsmap=None, attrib=None)
     return str(lxml_html.tostring(elem, encoding="unicode"))
+
+
+def is_empty_content_state(item: Dict, field: str) -> bool:
+    content_state = get_field_content_state(item, field)
+    return content_state is None or not any([block.get("text", "").strip() for block in content_state["blocks"]])
+
+
+def copy_fields(source: Dict, dest: Dict, ignore_empty=False):
```

Comment:
Same, please as a docstring here. Even if we can guess with names, it would be useful to know what `ignore_empty` does exactly (is it that key is missing, set to `None` or something else?).